bip-0002.mediawiki

```diff
@@ -133,31 +133,46 @@ Will BIP comments be censored or limited to particular participants/"experts"?
 
 ==BIP licensing==
 
-New BIPs may be accepted with the following licenses:
-
 ===Specification===
 
+New BIPs may be accepted with the following licenses. Each new BIP must identify at least one acceptable license in its preamble. The License header in the preamble must be placed after the Created header. Each license must be referenced by their respecitve abbreviation given below.
```

Comment:
There is a strict order defined in https://github.com/bitcoin/bips/blob/master/bip-0001.mediawiki#bip-header-preamble so let's jut keep a strict order.